docs/reference/commandline/plugin_enable.md

```diff
@@ -21,7 +21,8 @@ Usage:  docker plugin enable PLUGIN
 Enable a plugin
 
 Options:
-      --help   Print usage
+      --help          Print usage
+      --timeout int   HTTP client timeout (in seconds)
```

Comment:
There's more options that don't need sub-second precision, perhaps we should include it in a UX review (I do understand your point though)